File: isomorphic_strings.py
def isomorphic_strings(s, t):
    #write code here
    temp={}
    for i in range(len(s)):
        if s[i] not in temp:
            temp[s[i]]=t[i]
        
        if temp[s[i]]!=t[i]:
            return False
    
    return True

File: isomorphic_strings.js
const checkIsomorphic = function(s, t) {
    //write code here
    let temp = {};

    for (let i = 0; i < s.length; i++) {
        if (!temp[s[i]]) {
            temp[s[i]] = t[i];
        }

        if (temp[s[i]] != t[i]) {
            return false;
        }
    }
    return true;
}